projects/canopy/src/lib/card/card-hero-img/card-hero-img.component.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+
+import { LgCardHeroImageComponent } from './card-hero-img.component';
+
+describe('LgCardHeroImageComponent', () => {
+  let component: LgCardHeroImageComponent;
+  let fixture: ComponentFixture<LgCardHeroImageComponent>;
+
+  beforeEach(waitForAsync(() => {
+    TestBed.configureTestingModule({
+      declarations: [ LgCardHeroImageComponent ],
+    }).compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(LgCardHeroImageComponent);
+    component = fixture.componentInstance;
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+
+  it('should set the correct class if it has an image URL', () => {
```

Comment:
nit: I personally would just test the class function without involving the DOM

Reply:
Thanks @eduardh-landg, I see what you mean with this suggestion but the way Alex is done it is consistent with the way the rest of the tests are done in this project, so I think it's best to leave as-is for consistency :)

Reply:
As Owen mentioned, its more of a 'consistency' thing as those kind of test are all over in Canopy and done with this approach :)